### DISCUSSION TITLE
頁面全白

### DISCUSSION THREAD

**Discussion by 1030Tim** [MaLuns/hexo-theme-async#129] [Q&A 疑难杂症]
您好我按照了文本步驟至github發布後頁面顯示全白且文本是可以複製的，但在本地顯示是可以的，我該怎麼解決此問題
https://1030tim.github.io/my-web.github.io/

![image](https://github.com/user-attachments/assets/04d5c5e7-0e74-4059-b7b6-8048f74a414c)

<img width="1491" alt="image" src="https://github.com/user-attachments/assets/05768201-a404-4f07-b752-a50e41ed8173">

**Comment by MaLuns** [comment]
![image](https://github.com/user-attachments/assets/61c23e17-c492-4829-b099-37bbf5736400)

![image](https://github.com/user-attachments/assets/d96f04f3-1936-49b8-a1e4-0f73c55413e5)

看起来像是你自定义的代码报错呢

**Comment by MaLuns** [comment]
![image](https://github.com/user-attachments/assets/5a6e6498-1f9e-4262-a694-dfe3d67b2604)

主题相关静态文件都没有获取到, 检查下你的托管 git 上生成文件看看

**Comment by 1030Tim** [comment]
我看他生成的靜態html沒有寫到主題css的連結是這個問題嗎（抱歉打擾新手一枚

白云苍狗 ***@***.***>於 2024年8月26日 週一，下午1:35寫道：

**Comment by 1030Tim** [comment]
是找不到js嗎
我要如何在hexo或GitHub內調整
￼

**Reply by MaLuns** [reply]
js 和 css 文件都没找到呢, 你本地都构建下

**Comment by 1030Tim** [comment]
還是要調整_config.yml的root

**Reply by MaLuns** [reply]
应该和配置没有关系, 使用了主题生成时候就会有

你有改主题源码吗? 你看看你生成文件有没有这个
![image](https://github.com/user-attachments/assets/e3454edf-531c-4db5-9c9b-0111b9de8d42)

**Reply by MaLuns** [reply]
> 還是要調整_config.yml的root
> […](#)
> Juhn Chen ***@***.***> 於 2024年8月26日 13:52 寫道： 我看他生成的靜態html沒有寫到主題css的連結是這個問題嗎（抱歉打擾新手一枚 白云苍狗 ***@***.*** ***@***.***>>於 2024年8月26日 週一，下午1:35寫道： > > image.png (view on web) <https://github.com/user-attachments/assets/5a6e6498-1f9e-4262-a694-dfe3d67b2604> > 主题相关静态文件都没有获取到, 检查下你的托管 git 上生成文件看看 > > — > Reply to this email directly, view it on GitHub <[#129 (comment)](https://github.com/MaLuns/hexo-theme-async/discussions/129#discussioncomment-10447256)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A6C54WD5PCAXETFSFCQBEJDZTK5CTAVCNFSM6AAAAABNDIDMKGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANBUG4ZDKNQ>. > You are receiving this because you authored the thread. >

把使用 async 的 _config.yml 的 root 设置成 /my-web.github.io/ 试试呢, 你是把多个 hexo 一起部署然后是在不同子目录是吧? 如果是这样不同站点 root 要设置成对应子目录

加上 /my-web.github.io/ 静态文件是可以访问的

![image](https://github.com/user-attachments/assets/15c767ac-9852-4887-9d99-79a26cfc4005)

**Comment by 1030Tim** [comment]
> 都沒有
我該怎麼辦
是hexo沒有安裝好嗎
￼
￼

**Comment by 1030Tim** [comment]
我新創的所有檔案都沒有

**Reply by MaLuns** [reply]
> 我新創的所有檔案都沒有
> […](#)
> 白云苍狗 ***@***.***> 於 2024年8月26日 15:02 寫道： 应该和配置没有关系, 使用了主题生成时候就会有 你有改主题源码吗? 你看看你生成文件有没有这个 image.png (view on web) <https://github.com/user-attachments/assets/e3454edf-531c-4db5-9c9b-0111b9de8d42> — Reply to this email directly, view it on GitHub <[#129 (reply in thread)](https://github.com/MaLuns/hexo-theme-async/discussions/129#discussioncomment-10447954)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A6C54WGJLVSCIJKRLTCNUPLZTLHHJAVCNFSM6AAAAABNDIDMKGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANBUG44TKNA>. You are receiving this because you authored the thread.

看你其他页面还是原来的主题, 首页是新的主题,看起来像是混合到一起了? 生成的有缓存? 重新建一个新的呢?也会这样吗

![image](https://github.com/user-attachments/assets/23a11682-c6b3-4518-ab83-aca26c223c22)

**Reply by 1030Tim** [reply]
這個不是自己用hexo的而是別人html改的

**Comment by 1030Tim** [comment]
我一共開了5五個檔案
分別用了原生的主題與async
原生的是畫面完全跑版沒有圖片
async的主題是全部畫面都看不到（但感覺有跑版）
我有清緩存了甚至連hexo都重裝都沒有解決問題
￼

**Reply by MaLuns** [reply]
方便发下源码吗

**Comment by 1030Tim** [comment]
整個檔案嗎

**Comment by 1030Tim** [comment]
附件可使用到2024年9月25日
這是套用async的


> 白云苍狗 ***@***.***> 於 2024年8月26日 20:41 寫道：
> 
> 
> 两个主题一起使用的?
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/MaLuns/hexo-theme-async/discussions/129#discussioncomment-10451438>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A6C54WHJYXZ7V5X5F34ARLTZTMO7HAVCNFSM6AAAAABNDIDMKGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANBVGE2DGOA>.
> You are receiving this because you authored the thread.
> 
按一下來下載
 <https://www.icloud.com/attachment/?u=https%3A%2F%2Fcvws.icloud-content.com%2FB%2FASdK0-7bsUQgr8Dkgr8qZPhsPRlbAaz8NMfcE8IqbVB974lNxil8Wfnc%2F%24%7Bf%7D%3Fo%3DAuExalwW79MQSa2sdC9v-z66WmpFpopBEVpv5qKtOVQg%26v%3D1%26x%3D3%26a%3DCAogSdD5gSZLpA9WP29FdPKAJkbKW02En4o6OICqlhZOcH0SehD50_b1mDIY-ePxyaIyIgEAKgkC6AMA_2_KmLVSBGw9GVtaBHxZ-dxqJ9oR3nTsCXeLxpvw0vijaHXOwCItgqCU__DCFvOb1h4qw814tr2y8HInKIO2-FXxe62254C5Ne99IUFZSfJUauZ-3nmv0bnDAP1KE5Glz8ks%26e%3D1727268680%26fl%3D%26r%3D1A6C5CAC-7479-47C6-BE4A-EBCD2DB45B04-1%26k%3D%24%7Buk%7D%26ckc%3Dcom.apple.largeattachment%26ckz%3DB70A0E2F-C0D4-402F-9D80-6E36558905B9%26p%3D153%26s%3DKg5uW7X9Z9Pj9vV7VlKO81GDhUk&uk=h8yEmNPYYG1-9hNxHRB8pQ&f=my-blog.zip&sz=20609067>my-blog.zip
20.6 MB

**Comment by 1030Tim** [comment]
可以訪問但就是全部都白色的
多個hexo一起部署是什麼意思？
我有改了root但是還是一樣是白的
還是您可以提供我源碼我部署上去一次，如果沒問題是我源碼的問題有問題是git的問題？？

**Comment by 1030Tim** [comment]
我在本地用sudo hexo server 是完全可以顯示出來的但是轉成靜態網站及部署就出問題

**Comment by MaLuns** [comment]
就比如说 async 的示例站点是放在 /demosite/ 目录下的, 所以 root 设置为 demosite , 这样生成的时候静态文件会加上 demosite 
![image](https://github.com/user-attachments/assets/3a245d4d-5310-4c8d-9de5-74b37120f828)

但是你的 async 是放在 /my-web.github.io/ 下, 静态资源生成没有拼接上 

![image](https://github.com/user-attachments/assets/c6d543b7-462e-47a8-9338-a20a2a14510d)

**Comment by 1030Tim** [comment]
Hexo檔案可能缺失嗎

Juhn Chen ***@***.***>於 2024年8月26日 週一，下午9:48寫道：

**Comment by 1030Tim** [comment]
macOS有影響嗎

**Reply by MaLuns** [reply]
> macOS有影響嗎
> […](#)
> Juhn Chen ***@***.***> 於 2024年8月27日 08:19 寫道： Hexo檔案可能缺失嗎 Juhn Chen ***@***.*** ***@***.***>>於 2024年8月26日 週一，下午9:48寫道： > 我在本地用sudo hexo server 是完全可以顯示出來的但是轉成靜態網站及部署就出問題 > > >> Juhn Chen ***@***.*** ***@***.***>> 於 2024年8月26日 21:46 寫道： >> >> 可以訪問但就是全部都白色的 >> 多個hexo一起部署是什麼意思？ >> 我有改了root但是還是一樣是白的 >> 還是您可以提供我源碼我部署上去一次，如果沒問題是我源碼的問題有問題是git的問題？？ >> >> >>> 白云苍狗 ***@***.*** ***@***.***>> 於 2024年8月26日 21:34 寫道： >>> >>> >>> 還是要調整_config.yml的root >>> … <> >>> Juhn Chen @.> 於 2024年8月26日 13:52 寫道： 我看他生成的靜態html沒有寫到主題css的連結是這個問題嗎（抱歉打擾新手一枚 白云苍狗 @. @.***>>於 2024年8月26日 週一，下午1:35寫道： > > image.png (view on web) https://github.com/user-attachments/assets/5a6e6498-1f9e-4262-a694-dfe3d67b2604 > 主题相关静态文件都没有获取到, 检查下你的托管 git 上生成文件看看 > > — > Reply to this email directly, view it on GitHub <#129 (comment) <[#129 (comment)](https://github.com/MaLuns/hexo-theme-async/discussions/129#discussioncomment-10447256)>>, or unsubscribe https://github.com/notifications/unsubscribe-auth/A6C54WD5PCAXETFSFCQBEJDZTK5CTAVCNFSM6AAAAABNDIDMKGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANBUG4ZDKNQ. > You are receiving this because you authored the thread. > >>> >>> 把使用 async 的 _config.yml 的 root 设置成 /my-web.github.io/ <http://my-web.github.io/> 试试呢, 你是把多个 hexo 一起部署然后是在不同子目录是吧? 如果是这样不同站点 root 要设置成对应子目录 >>> >>> 加上 /my-web.github.io/ <http://my-web.github.io/> 静态文件是可以访问的 >>> >>> image.png (view on web) <https://github.com/user-attachments/assets/15c767ac-9852-4887-9d99-79a26cfc4005> >>> — >>> Reply to this email directly, view it on GitHub <[#129 (reply in thread)](https://github.com/MaLuns/hexo-theme-async/discussions/129#discussioncomment-10452098)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A6C54WHPR7TVPSBGQQBJLCDZTMVHPAVCNFSM6AAAAABNDIDMKGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANBVGIYDSOA>. >>> You are receiving this because you commented. >>> >> >

没影响的

**Comment by MaLuns** [comment]
你这个应该就是上面说的那样,导致生成路径不对,然后无法加载样式那些

**Comment by 1030Tim** [comment]
要如何解決

**Comment by MaLuns** [comment]
按照上面的设置root 就可以啊

**Comment by 1030Tim** [comment]
￼要如何將貼文圖片替換掉
